xc/xc7/toolchain_wrappers/symbiflow_synth

```diff
@@ -92,13 +87,15 @@ if [ ${#VERILOG_FILES[@]} -eq 0 ]; then
 	exit 1
 fi
 
+DATABASE_DIR=`prjxray-config`
```

Comment:
This should be:
```
: $(DATABASE_DIR:=$(prjxray-config)
```
So that overriding the DATABASE_DIR is possible?

Reply:
Yes. makes sense, done